wrappers/kmip/README.md

```diff
@@ -0,0 +1,54 @@
+# KMIP wrapper
+
+Provides integration with KMIP v1.2 up to v1.4 compatible servers.
+
+## Settings
+| Environment variable   | Required | Default          | Description                                              |
+| ---------------------- | -------- | ---------------- | -------------------------------------------------------- |
+| KMIP_WRAPPER_KEY_ID    | yes      |                  | Wrapping key ID                                          |
+| KMIP_ENDPOINT          | yes      |                  | KMIP endpoint address and port                           |
+| KMIP_CA_CERT           | no       |                  | Server CA certificate                                    |
+| KMIP_CLIENT_CERT       | yes      |                  | Client authentication TLS certificate                    |
+| KMIP_CLIENT_KEY        | yes      |                  | Client TLS private key                                   |
+| KMIP_SERVER_NAME       | no       |                  | Used to verify the hostname on the returned certificates |
+| KMIP_TIMEOUT           | no       | 10               | KMIP operation timeout in seconds                        |
+| KMIP_ENCRYPT_ALG       | no       | AES_GCM          | Encryption algorithm to use                              |
+| KMIP_TLS12_CIPHERS     | no       | Golang's default | Coma separated list of TLS 1.2 ciphers to allow          |
+
+## KMIP server requirements
+- Must support one of the following KMIP protocol version: v1.2, v1.3 or v1.4.
+- Must support Encrypt & Decrypt operations
+- Must support AES symmetric keys
+- Must support one of the following cryptographic parameters combination
+
+## Supported KMIP Encryption Algorithms
+
+### AES_GCM (recommended)
+Should be supported on most KMIP 1.4 compatible servers.
+Support on KMIP 1.2 and 1.3 vary depending on vendors.
+
+#### KMIP Cryptographic Parameters
+| Parameter               | Value    |
+| ----------------------- | -------- |
+| Cryptographic Algorithm | AES      |
+| Block Cipher Mode       | GCM      |
+| Padding Method          | PKCS#5   |
+| Tag length              | 16 bytes |
+| Nonce length            | 12 bytes |
+
+### AES_CBC_PKCS
```

Comment:
I'm happy dropping support for AES-CBC like we did in PKCS#11 interfaces: #14

(Description is misleading, only GCM and OAEP made the final cut). 

Especially since this is a newer integration, it is 2025, we don't need to support CBC and PKCS#1v1.5.

Reply:
I agree. I only kept it for compatibility with the base PR.
I removed it, and Iin the meantime also added support for RSA OAEP